### DISCUSSION TITLE
Surface Mesh Generation

### DISCUSSION THREAD

**Discussion by DrHybris** [mdolab/dafoam#480] [General]
I am seeking online resources to learn about generating surface meshes for components such as wings and fuselages in computational fluid dynamics (CFD) applications. Specifically, I have noticed that many technical papers focusing on optimization utilize tools like PyGeo, PySurf, and IDWarp for surface mesh generation. However, the available tutorials often only cover volumetric mesh generation using software like ICEM CFD. I would like to request recommendations for videos or web pages that provide detailed instructions on generating surface meshes for wings and fuselages, preferably using open-source tools. If possible, I would appreciate these resources including direct links to access the relevant information.

**Comment by friedenhe** [comment]
If you want to use pyHyp to generate structured volume meshes, we suggest you use ICEM-CFD to generate the surface mesh. If you use unstructured meshes, we suggest you use Pointwise. They have very good tutorials on how to generate both surface and volume meshes. We are not aware of a good open-source mesh generation tool that works well for our cases.
